contracts/ika_dwallet_2pc_mpc/sources/coordinator_inner.move

```diff
@@ -4697,6 +5006,26 @@ public(package) fun set_paused_curves_and_signature_algorithms(
     self.support_config.set_paused(paused_curves, paused_signature_algorithms, paused_hash_schemes);
 }
 
+/// Sets the global presign config.
+///
+/// ### Parameters
+/// - **`curve_to_signature_algorithms_for_dkg`**: Map of curves to signature algorithms for dWallets that are created via DKG.
+/// - **`curve_to_signature_algorithms_for_imported_key`**: Map of curves to signature algorithms for dWallets that are created via imported key.
+public(package) fun set_global_presign_config(
```

Comment:
why is this a new function? it should have been existed no?

Reply:
We don't have that, and I recreated all the global presign staff, and the old value is deprecated